python/lsst/ts/m2gui/controltab/tab_alarm_warn.py

```diff
@@ -30,9 +30,15 @@
     QTableWidgetItem,
```

Comment:
A general comment: Qt naming uses suffixes to qualify type, e.g. DockWidget, not WidgetDock. It would be much better to follow the suit, and name your artefacts using the same strategy - e.g. AlarmWarnDock or AlarmWarnWidget, instead of TabDefault DefaultWidget, DefaultDock or DefaultTab

Reply:
I prefer to put the common part of name first because it is easier to find the related classes/functions in `iPython` (use `tab` twice). Thanks!